lib/modules/showImages.js

```diff
@@ -249,6 +251,21 @@ module.options = {
 		value: true,
 		description: 'Show the site logo and name after embedded content.',
 	},
+	expandoCommentRedirects: {
+		type: 'enum',
+		value: 'expando',
+		values: [{
+			name: 'Do nothing',
+			value: 'nothing',
+		}, {
+			name: 'Create expandos',
+			value: 'expando',
+		}, {
+			name: 'Create expandos and rewrite the link',
```

Comment:
This is probably too vague but I'm not sure how to concisely say "change the link to point to the image instead of the comments"